BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.m

```diff
@@ -373,11 +396,17 @@ - (void)setAccessTokenDelegate:(id<BOXAPIAccessTokenDelegate>)accessTokenDelegat
 
 + (void)oneTimeSetUpInAppToSupportBackgroundTasksWithDelegate:(id<BOXURLSessionManagerDelegate>)delegate rootCacheDir:(nonnull NSString *)rootCacheDir completion:(void (^)(NSError *error))completionBlock;
 {
+    [BOXContentClient defaultClient].tempCacheDir = rootCacheDir;
+    
+    rootCacheDir = [rootCacheDir stringByAppendingString:BOXSessionManagerCacheClientFolder];
     [[BOXURLSessionManager sharedInstance] oneTimeSetUpInAppToSupportBackgroundTasksWithDelegate:delegate rootCacheDir:rootCacheDir completion:completionBlock];
 }
 
 + (void)oneTimeSetUpInExtensionToSupportBackgroundTasksWithDelegate:(id<BOXURLSessionManagerDelegate>)delegate rootCacheDir:(nonnull NSString *)rootCacheDir sharedContainerIdentifier:(NSString *)sharedContainerIdentifier completion:(void (^)(NSError *error))completionBlock;
 {
+    
+    [BOXContentClient defaultClient].tempCacheDir = rootCacheDir;
```

Comment:
assigning tempCacheDir = rootCacheDir might give the assumption that tempCacheDir stores the same value as rootCacheDir.

We could have a setUpTempCacheDir:rootCacheDir method instead? or compose [rootCacheDir stringByAppendingPathComponent:BOXContentClientBackgroundTempFolder] to assign to tempCacheDir?

it's more readable that the